src/layers/domain/core/fhir_transform.py

```diff
@@ -0,0 +1,57 @@
+from typing import TypeVar
+
+from domain.core.product import ProductTeam
+from domain.core.root import Root
+from domain.fhir.r4 import cpm_model
+from domain.fhir.r4.models import Organization
+from domain.fhir.r4.strict_models import Organization as StrictOrganization
+from pydantic import BaseModel
+
+fhir_type = TypeVar("fhir_type")
+
+
+def create_product_team_from_fhir_json(fhir_json: dict, **kwargs) -> ProductTeam:
+    fhir_model = create_fhir_model_from_fhir_json(
```

Comment:
suggest changes to names to make clear that we're converting a product team from a fhir org:

* `create_product_team_from_fhir_json` --> `create_product_team_from_fhir_org_json`
* `fhir_model` to `fhir_org` 
* `fhir_json` to `fhir_org_json`